### PR TITLE
Add document on stateless concept

### PR DESCRIPTION
**Pull request by puneetse**
This PR re-adds a document on the stateless concept closes https://github.com/clearlinux/clear-linux-documentation/issues/545 

The rendered version with images can be viewed onat: https://github.com/puneetse/clear-linux-documentation/blob/add/ps-stateless/source/clear-linux/concepts/stateless.rst. Please excuse the large image size, it is scaled down but GH does not understand that RST parameter.

### CODE REVIEW

File: source/clear-linux/concepts/stateless.rst [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+
+Overriding configurations
+=========================
+
+If a configuration needs to be changed, the appropriate file should be
+modified by the user under :file:`/etc/`. If the configuration file does not
+already exist, it can be created in the appropriate location.
+
+User defined configuration files should contain the minimal set of desired
```
Comment (puneetse) [inline source/clear-linux/concepts/stateless.rst]: >User defined configuration files should contain the minimal set of desired
changes and rely on default configuration for the rest. 

@ahkok I'm particularly interested if you are in agreement with this statement for CL..

Comment (ahkok) [reply source/clear-linux/concepts/stateless.rst]: Pretty sure I wrote this.

**Comment by puneetse** [comment]
@krisdale or @mvincerx can you please provide a final review and merge if no other blockers?

**Comment by mvincerx** [comment]
LGTM. Thanks @puneetse.

**Pull request merged**